app/routes.php

```diff
@@ -455,11 +457,15 @@ function ( Application $app, Request $request ) use ( $ffFactory ) {
 	}
 );
 
-$app->get( '/', function ( Application $app ) {
-	return $app->handle(
-		Request::create( '/page/DonationForm', 'GET' ),
-		HttpKernelInterface::SUB_REQUEST
-	);
+$app->get( '/', function ( Application $app, Request $request ) use ( $ffFactory ) {
```

Comment:
Is this change really needed? Would't it be handled by the subrequest to add donation?